connect/api/src/main/java/org/apache/kafka/connect/sink/SinkConnector.java

```diff
@@ -34,4 +34,8 @@
      */
     public static final String TOPICS_CONFIG = "topics";
 
+    public SinkConnectorContext context() {
```

Comment:
Doesn't this need an `@Override`? Would be great to have JavaDoc here as well, since it is changing the return type defined by the inherited method.